src/Utils/llm_utils.py

```diff
@@ -91,6 +91,8 @@ def _handle_chat_nvidia(
             result = llm_chain.invoke(input)
             if result is not None:
                 return result
+            else:
+                last_exception = "No exception was raised, but the response is None"
```

Comment:
When can this happens? Maybe add a little elaboration because this message will not give any clue.